File: user-app/src/main/java/com/github/dkurata38/liblio/user_app/UserApplication.java
package com.github.dkurata38.liblio.user_app;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Import;
import com.github.dkurata38.liblio.application.ApplicationConfig;
import com.github.dkurata38.liblio.infrastructure.InfrastructureConfig;

@SpringBootApplication
@Import(value = {InfrastructureConfig.class, ApplicationConfig.class})
public class UserApplication {
    public static void main(String[] args) {
        SpringApplication.run(UserApplication.class, args);
    }
}

File: user-app/src/main/kotlin/com/github/dkurata38/liblio/user_app/UserApplication.kt
package com.github.dkurata38.liblio.user_app

import com.github.dkurata38.liblio.application.ApplicationConfig
import com.github.dkurata38.liblio.infrastructure.InfrastructureConfig
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Import

@SpringBootApplication
@Import(ApplicationConfig::class, InfrastructureConfig::class)
class UserApplication

fun main(args: Array<String>) {
    runApplication<UserApplication>(*args)
}
